LandLord-legacy/src/main/java/biz/princeps/landlord/manager/MobManager.java

```diff
@@ -86,9 +86,9 @@ public IMob valueOf(String name) {
 
     public static class Mob implements IMob {
 
-        final EntityType t;
-        final int egg;
-        final String permission;
+        EntityType t;
+        int egg;
+        String permission;
```

Comment:
Those are fields, that's okay to be final

Reply:
Nuked them without considering such variables, ty!